pkg/rbac/userData.go

```diff
@@ -111,6 +112,24 @@ func (cache *Cache) GetUserData(ctx context.Context,
 
 }
 
+func (cache *Cache) GetUserData(ctx context.Context) (*UserData, error) {
```

Comment:
```suggestion
// Get a static copy of the current user data. It will use cached data if valid or refresh if needed.
func (cache *Cache) GetUserData(ctx context.Context) (*UserData, error) {
```

Reply:
Done